### DISCUSSION TITLE
problem with mariadb

### DISCUSSION THREAD

**Discussion by acron097** [photoprism/photoprism#4392] [Q&A]
![screenshot](https://github.com/photoprism/photoprism/assets/168218331/73c4f642-4474-441a-9047-9b9f2ebc4638)

I tried to install it with docker-compose, but I ran into an error, which you can see in the picture. The photoprism was running on a virtual machine with ubuntu 22 server. I tried accessing it from Chrome and Firefox browsers as follows: http://internalipofthevirtualmachine:2342 . Later I changed the port from "2342:2342" to "80:2342", but that didn't help either. Of course I restarted the containers several times, including mariadb. The solutions I have found do not work and are outdated. Thanks in advance for your help.

**Comment by graciousgrey** [comment]
You can find some related discussions here:

- https://github.com/photoprism/photoprism/discussions/3834
- https://github.com/photoprism/photoprism/issues/67#issuecomment-752894229
